src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/Checker.java

```diff
@@ -214,37 +214,25 @@ public Void scan(DocCommentTree tree, TreePath p) {
         foundReturn = false;
         hasNonWhitespaceText = false;
 
-        switch (p.getLeaf().getKind()) {
+        implicitHeadingRank = switch (p.getLeaf().getKind()) {
```

Comment:
(observation) Since _rank_ is a rather unusual word to see, I explored this a bit. Numerals in H1, H2, H3, H4, H5, and H6 were somewhat implicitly referred to as _heading levels_ by HTML4, and indeed became referred to as _ranks_ in HTML5.

Reply:
yes, that is where I found the term.